src/App.js

```diff
@@ -1,23 +1,24 @@
+import React from 'react';
 import './App.css';
-import { Route, BrowserRouter as Router, Routes } from 'react-router-dom';
-import Header from './components/Header/Header';
-import Mission from './components/Mission/Mission';
-import Profile from './components/Profile/Profile';
-import Rocket from './components/Rocket/Rocket';
+
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Rockets from './components/Rocket/Rockets';
+import Navbar from './components/Navbar';
 
 function App() {
   return (
-    <>
-      <Router>
-        <Header />
+    <div className="App">
+      <BrowserRouter>
+        <Navbar />
         <Routes>
-          <Route path="/" element={<Rocket />} />
-          <Route path="mission" element={<Mission />} />
-          <Route path="profile" element={<Profile />} />
-          <Route path="*" element={<h2>Something Went wrong!</h2>} />
+          <Route path="/" element={<Rockets />} />
         </Routes>
-      </Router>
-    </>
+      </BrowserRouter>
```

Comment:
Please try to Add the navigation bar in this project because it will allow users to navigate from the rocket to the mission and the profile components. As I can see the navigation of the project is the header components.

Reply:
@Kaghenimbale ok thanks, I will do that now.

Reply:
I don't think it is really needed at this stage since you already implement it on your side and it will appear after the final merge. But let me do it yeah.